code/modules/jobs/job_types/curator.dm

```diff
@@ -56,6 +56,8 @@
 
 	accessory = /obj/item/clothing/accessory/pocketprotector/full
 
+	skillchips = list(/obj/item/skillchip/xenoarch_magnifier) // SKYRAT EDIT: for the xenoarch thing
```

Comment:
This could be done modularly, just so it doesn't cause conflicts (so having it in its own files in the `master_files` folder.